### PR TITLE
Demos: supporting testing across multiple themes

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

File: apps/demos/utils/visual-tests/matrix-test-helper.js [hunk 1]
```diff
@@ -54,6 +69,33 @@ export function getChangedFiles() {
...
+export function globalReadFrom(basePath, relativePath, mapCallback) {
+  const absolute = join(basePath, relativePath);
+  if (existsSync(absolute)) {
+    const result = readFileSync(absolute, 'utf8');
+    return (mapCallback && result && mapCallback(result)) || result;
+  }
+  return null;
+}
+
+export function changeTheme(dirName, relativePath, demoPath, theme) {
```
Comment (iBat) [inline apps/demos/utils/visual-tests/matrix-test-helper.js]: 3 starting params looks like overkill. As I see you always use together at least 1st and 2nd. Why dont change it with, say,  (demosRoot, demoPath, theme) and join some paths in a callers.

File: apps/demos/utils/visual-tests/matrix-test-helper.js [hunk 2]
```diff
@@ -54,6 +69,33 @@ export function getChangedFiles() {
...
+  if (!theme || theme === THEME.generic) {
+    return;
+  }
+
+  const indexFilePath = join(dirName, `${relativePath}${demoPath}/index.html`);
```
Comment (iBat) [inline apps/demos/utils/visual-tests/matrix-test-helper.js]: When you use `path.join` - no need to build strings. All strings anyway will be splitted by slashes/backslashes and rebuilded using `path.sep`. 
```suggestion
  const indexFilePath = join(dirName, relativePath, demoPath, 'index.html');
```

Comment (EugeniyKiyashko) [reply apps/demos/utils/visual-tests/matrix-test-helper.js]: https://github.com/DevExpress/DevExtreme/pull/26748/commits/5099ddceea7e349f16bf267aaa3e4127e5ef32bf

File: apps/demos/utils/visual-tests/matrix-test-helper.js (hunk 1 above)
Comment (EugeniyKiyashko) [reply apps/demos/utils/visual-tests/matrix-test-helper.js]: https://github.com/DevExpress/DevExtreme/pull/26748/commits/5099ddceea7e349f16bf267aaa3e4127e5ef32bf

File: .github/workflows/demos_visual_tests.yml [hunk 3]
```diff
@@ -72,12 +72,13 @@ jobs:
       fail-fast: false
       matrix:
         STRATEGY: [screenshots, accessibility]
+        THEME: ['generic.light', 'material.blue.light', 'fluent.blue.light']
```
Comment (marker-dao) [inline .github/workflows/demos_visual_tests.yml:75]: Why we are testing only light themes?

**Review by iBat — APPROVED** [review]

File: .github/workflows/demos_visual_tests.yml (hunk 3 above)
Comment (EugeniyKiyashko) [reply .github/workflows/demos_visual_tests.yml:75]: At the moment, yes. If there is a need to add other color schemes, we can do it later in separate pull request.

**Review by marker-dao — APPROVED** [review]

**Review by andrewmakarov — APPROVED** [review]

**Pull request merged**